src/main/java/com/alex/eshop/restcontroller/UserController.java

```diff
@@ -18,8 +18,14 @@ public UserController(UserService userService) {
     }
 
     @PreAuthorize("hasRole('ROLE_ADMIN')")
-    @GetMapping("users/{Uuid}")
-    public UserRepresentation getUser(@PathVariable String Uuid) {
-        return userService.getUserRepresentation(Uuid);
+    @GetMapping("/users/{Uuid}")
+    public UserDTO getUser(@PathVariable String Uuid) {
+        return userService.getUser(Uuid);
+    }
+
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @PostMapping("/users")
```

Comment:
You have path ("`/api/users/register`") permitted for all in `WebSecurityConfig`. It supposed to be used for this function